openpdf/src/main/java/com/lowagie/text/pdf/PdfPTable.java

```diff
@@ -1031,8 +1032,8 @@ public void setHeaderRows(int headerRows) {
      *
      * @return    an <CODE>ArrayList</CODE>
      */
-    public ArrayList getChunks() {
-        return new ArrayList();
+    public java.util.ArrayList<Element> getChunks() {
```

Comment:
Why a FQCN here? Could be only
```suggestion
    public ArrayList<Element> getChunks() {
```

Reply:
@asturio : I agree with you. But I'm fixing some of these FQCN in my last PR #228. I can either change it here or in the final PR.
What do you think?

Reply:
Final PR is ok